src/protocol/bolt/src/main/java/org/apache/jmeter/protocol/bolt/config/BoltConnectionElementBeanInfo.java

```diff
@@ -44,6 +44,9 @@ public BoltConnectionElementBeanInfo() {
         propertyDescriptor = property("password", TypeEditor.PasswordEditor);
         propertyDescriptor.setValue(NOT_UNDEFINED, Boolean.TRUE);
         propertyDescriptor.setValue(DEFAULT, "");
+        propertyDescriptor = property("database");
+        propertyDescriptor.setValue(NOT_UNDEFINED, Boolean.TRUE);
+        propertyDescriptor.setValue(DEFAULT, "neo4j");
```

Comment:
IMO, there should be no default value, as 3.5 server versions don't have the concept of databases, and in 4.x versions, the default Neo4j database can be changed in the server configuration file.